File: nelio-alves-algoritmo-e-logica-de-programacao/c/exercicios/numeros_pares.c
#include <stdio.h>

int main(){
    int N, cont;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    int vet[N];

    for (int i = 0; i < N; i++) {
        printf("Digite um numero: ");
        scanf("%d", &vet[i]);
    }

    cont = 0;
    printf("\nNUMEROS PARES:\n");
    for (int i = 0; i < N; i++) {
        if (vet[i] % 2 == 0) {
            printf("%d  ", vet[i]);
            cont++;
        }
    }

    printf("\n\nQUANTIDADE DE PARES = %d\n", cont);

    return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/numeros_pares.py
N: int; cont: int

N = int(input("Quantos numeros voce vai digitar? "))

vet: [int] = [0 for x in range(N)]

for i in range(N):
	vet[i] = int(input("Digite um numero: "))

cont = 0
print("\nNUMEROS PARES:")
for i in range(N):
	if vet[i] % 2 == 0:
		print(f"{vet[i]}  ", end="")
		cont = cont + 1

print(f"\n\nQUANTIDADE DE PARES = {cont}")